lib/channel-access-token/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-7.0.0
\ No newline at end of file
+unset
```

Comment:
Unfortunately, this will be `unset`.  Maybe it's the packaging issue...
But it's not a big problem since the version number is available in pom.xml!